android/src/main/java/com/imagepicker/Options.java

```diff
@@ -18,7 +19,12 @@
             pickVideo = true;
         }
         includeBase64 = options.getBoolean("includeBase64");
-        videoQuality = options.getString("videoQuality");
+        
+        String videoQualityString = options.getString("videoQuality");
+        if(!TextUtils.isEmpty(videoQualityString) && videoQualityString.toLowerCase() != "high") {
```

Comment:
change
`videoQualityString.toLowerCase() != "high"`
to
`videoQualityString.toLowerCase().equals("low")` or `!videoQualityString.toLowerCase().equals("high")`